docs/serving/spec/knative-api-specification-1.0.md

```diff
@@ -1376,22 +1434,17 @@ Restrictions to the values of the field are noted in the Description column.
    </td>
    <td>int
 <br>
-(Required)
+(Optional)
 <br>
 Min: 0
 <br>
 Max: 100
    </td>
-   <td>The
-
-<a href="#request-routing">percentage of requests which should be allocated from
-the main Route domain name</a> to the specified <code>revisionName</code> or
-<code>configurationName</code>.
-
+   <td>The <code>percent</code> is optionally used to specify the percentage of requests which should be allocated from the main Route domain name to the specified <code>revisionName</code> or <code>configurationName</code>.
 <p>
-All <code>percent</code> values in <code>traffic</code> MUST sum to 100.
+To indicate that percetage based routing is to be used, at least one <code>traffic</code> section have a non-zero <code>percent</code> value, and all values MUST sum to 100. Note, a missing <code>precent</code> value implies zero.
```

Comment:
```suggestion
To indicate that percentage based routing is to be used, at least one <code>traffic</code> section MUST have a non-zero <code>percent</code> value, and all values MUST sum to 100. Note, a missing <code>precent</code> value implies zero.
```

I think you lost a "MUST/MAY/SHOULD". Also, a typo in "percentage".

Reply:
doi - fixed